### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by tysongach**
- Stop nesting `dt` and `dd` elements (it outputs overly specific selectors for no reason)
- Exclude Node modules from SCSS-Lint
- Clear margins for data list elements
- Consolidate input styles
  - Don't declare font family and size redundantly (currently set both on Line 20, as well as Line 28)
  - Include `textarea` in input styles

### CODE REVIEW

File: core/_forms.scss [hunk 1]
```diff
@@ -31,8 +32,6 @@ select[multiple] {
   border-radius: $base-border-radius;
   box-shadow: $form-box-shadow;
   box-sizing: border-box;
-  font-family: $base-font-family;
-  font-size: $base-font-size;
   margin-bottom: $small-spacing;
   padding: $base-spacing / 3;
```
Comment (whmii) [inline core/_forms.scss]: this should die

**Comment by whmii** [comment]
good

File: core/_forms.scss (hunk 1 above)
Comment (tysongach) [inline core/_forms.scss]: `padding: $base-spacing / 3;`?

Agree, but submit a separate PR.
